packages/core/src/browser/style/search-box.css

```diff
@@ -16,22 +16,31 @@
 
 .theia-search-box {
     position: absolute;
-    top: 0px;
-    right: 0px;
+    top: 4px;
```

Comment:
This value seems to be a little bit off, the filter should actually sit below the titlebar:

VSCode:

![image](https://user-images.githubusercontent.com/62660714/106628231-31eea300-653f-11eb-91cb-cb74cb946b0a.png)

This branch:

![image](https://user-images.githubusercontent.com/62660714/106628490-78440200-653f-11eb-95ec-af05c4da9f5a.png)

You can use the variable `--theia-view-container-title-height` to calculate the proper offset with the 4px padding

Reply:
Regarding this, we have a scenario that title will be shown only when the explorer have 2 views (Explorer & NPM Script). In the other case it wont be available.

**Reference without title**
<img width="524" alt="Screenshot 2021-02-03 at 1 15 07 PM" src="https://user-images.githubusercontent.com/13016475/106714501-f1b42100-6621-11eb-8d39-e0742d86a568.png">

I will make changes to support both the behaviour.

Reply:
To support both (with title and without title) behaviour with 4px, have made the parent container as **position:relative**, also verified there is no other impact due to that change.
I also tried with --theia-view-container-title-height, but in explorer without titlebar, search box is too down thats looking odd in UI.